manifests/server/config.pp

```diff
@@ -147,6 +147,17 @@
                   Exec['puppet_server_config-create_ssl_dir'],
                   ],
     }
+  } elsif $::puppet::server::ca_crl_sync {
+    # If not a ca AND sync the crl from the ca master
+    if $::server_facts['servername'] != undef {
```

Comment:
server_facts isn't enabled by default - this probably needs:

1. a defined() check to ensure the feature is on
1. a fail() if the feature is off and ca_crl_sync is enabled